ze/tracer_ze_helpers.include.c

```diff
@@ -1,8 +1,24 @@
+#ifdef THAPI_DEBUG
+#define TAHPI_LOG stderr
+#define THAPI_DBGLOG(fmt, ...) \
+ do { \
+   fprintf(TAHPI_LOG, "THAPI(%s:%d): " fmt "\n", __func__, __LINE__, __VA_ARGS__); \
+ } while(0)
+#define THAPI_DBGLOG_NO_ARGS(fmt) \
```

Comment:
Look like we can use https://stackoverflow.com/a/59044559/7674852, to remove the `NO_ARGS`

Reply:
`__VA_OPT__ (c++2a)`
It would not work in C mode with our current flags.

Reply:
Yes, it is an extension, so our strict flags will trigger. Let's revisit once we move to later C standards.